client/src/App.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { fireEvent, render, screen } from '@testing-library/react'
+import fakeKiosk from '../test/helpers/fakeKiosk'
+
+import App from './App'
+
+beforeEach(() => {
+  // we don't care about network errors logged to the console, they're crowding things
+  jest.spyOn(console, 'error').mockImplementation(() => {}) // eslint-disable-line @typescript-eslint/no-empty-function
+
+  window.location.href = '/'
+  window.kiosk = fakeKiosk()
+})
+
+it('basic navigation works', async () => {
+  const { getByText } = render(<App />)
+
+  await screen.findByText('Create New Election Definition')
```

Comment:
Why is this needed?

Reply:
Because there is enough asynchronous stuff going on (including that network call that fails) that if we don't wait for this, we fail tests.